app/components/EventItem/styles.css

```
@@ -6,6 +6,13 @@
   display: flex;
   justify-content: space-between;
   line-height: 1.3;
+  transition: var(--easing-fast);
+}
+
+.eventItemHover {
```

```suggestion
.eventItem:hover {
```

You can do this instead of using TypeScript!

---

➕